source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.importance.FanovaImportanceEvaluator.po

```diff
@@ -0,0 +1,168 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2018, Optuna Contributors.
+# This file is distributed under the same license as the Optuna package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Optuna 2.4.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-01-14 18:04-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: ../../source/reference/generated/optuna.importance.FanovaImportanceEvaluator.rst:2
+msgid "optuna.importance.FanovaImportanceEvaluator"
+msgstr ""
+
+#: of optuna.importance.FanovaImportanceEvaluator:1
+msgid "fANOVA importance evaluator."
+msgstr ""
+
+#: of optuna.importance.FanovaImportanceEvaluator:3
+msgid ""
+"Implements the fANOVA hyperparameter importance evaluation algorithm in "
```

Comment:
Would you translate these, or they are for another PR?

Reply:
Sorry for the delayed response. Yeah they will be in another PR.